x-pack/test_serverless/functional/test_suites/observability/dataset_quality/dataset_quality_privileges.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { getInitialTestLogs } from './data';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const PageObjects = getPageObjects([
+    'common',
+    'navigationalSearch',
+    'observabilityLogsExplorer',
+    'datasetQuality',
+    'svlCommonNavigation',
+    'svlCommonPage',
+  ]);
+  const testSubjects = getService('testSubjects');
+  const synthtrace = getService('svlLogsSynthtraceClient');
+  const find = getService('find');
+  const to = '2024-01-01T12:00:00.000Z';
+
+  describe('Dataset quality user privileges', function () {
+    this.tags(['failsOnMKI']);
+
+    before(async () => {
+      await PageObjects.svlCommonPage.loginWithRole('viewer');
+      await synthtrace.clean();
```

Comment:
Here also, adding a clean in the starting of the suite, makes it impossible to debug data spills